src/adapters/scene-stream-access-manager.ts

```diff
@@ -0,0 +1,61 @@
+import { AppComponents, AddSceneStreamAccessInput, ISceneStreamAccessManager, SceneStreamAccess } from '../types'
+import SQL from 'sql-template-strings'
+
+export async function createSceneStreamAccessManagerComponent({
+  database,
+  logs
+}: Pick<AppComponents, 'database' | 'logs'>): Promise<ISceneStreamAccessManager> {
+  const logger = logs.getLogger('scene-stream-access-manager')
+
+  async function addAccess(input: AddSceneStreamAccessInput): Promise<void> {
+    logger.debug('Adding stream access', { place_id: input.place_id })
+
+    await database.query(
+      SQL`UPDATE scene_stream_access 
+          SET active = false 
+          WHERE place_id = ${input.place_id} AND active = true`
+    )
+
+    const now = Date.now()
+
+    await database.query(
+      SQL`INSERT INTO scene_stream_access 
+          (id, place_id, streaming_key, streaming_url, created_at, active) 
+          VALUES 
+          (gen_random_uuid(), ${input.place_id}, ${input.streaming_key}, ${input.streaming_url}, ${now}, true)`
```

Comment:
In the definition for the `scene_stream_access`, you can set the `created_at` and `active` properties to be set as default: `NOW()` and `true`, which will avoid you from setting them here.